current_range.test.py

```diff
@@ -0,0 +1,14 @@
+#Test_current_ranges
+
+import unittest
+import identify_range_and_count_implementation as ir
+
+class CurrentRangeTest(unittest.TestCase):
+    def passing_test_range_and_count(self):
+        ir.identify_range_and_count([3, 3, 5, 4, 10, 11, 12])
```

Comment:
These tests dont have asserts. If you wrote them before writing the code, then you've only designed the function name and parameters, not the output.
E.g., are single values ignored or made into a range? Are the range and the counts separate or together?
First write a test, imagining a pure-function without the printf (returning a dictionary). Assert the values of the dictionary. Then write the code.